frontend/pages/404.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Center, Text, Divider } from '@chakra-ui/react';
+import {
+   DefaultLayout,
+   DefaultLayoutProps,
+   getLayoutServerSideProps,
+} from '@layouts/default';
+
+export default function Custom404(layoutProps: DefaultLayoutProps) {
+   return (
+      <DefaultLayout {...layoutProps}>
+         {
```

Comment:
Do these braces do anything?

Reply:
I thought I needed them for the `DefaultLayout` `children` parameter, but I removed it and it looks like they aren't necessary.